stdpopsim/cli.py

```diff
@@ -755,7 +755,7 @@ def run_simulation(args):
                 left = np.min(intervals)
                 right = np.max(intervals)
                 intervals_summary_str = f"[{left}, {right})"
-            else:
+            if intervals_summary_str is None:
```

Comment:
Oh dear, good catch. I guess we need some more tests for correctness of the CLI somewhere - I'll open an issue.